src/ViewInstance.js

```diff
@@ -503,7 +504,7 @@ class ViewInstance extends React.Component {
           </IfInterface>
 
           {
-            canEditDeriveMARCRecord && (
+            (canEditDeriveMARCRecord || canViewMARCSource) && (
```

Comment:
We only need to check for `canViewMARCSource`. Permission to view MARC record is included in permissions to edit and derive, so there's no need to check twice here

Reply:
Thanks, you are right, I fixed it.